include/Core/Base/Mixin/Renderer.php

```diff
@@ -558,12 +558,25 @@ protected function render_delete_settings( $hide_trash = false ) {
 	 * Render limit settings.
 	 *
 	 * @param string $item_type Item Type to be displayed in label.
+	 *
 	 */
 	protected function render_limit_settings( $item_type = '' ) {
-		if ( empty( $item_type ) ) {
-			$item_type = $this->item_type;
-		}
-		bd_render_limit_settings( $this->field_slug, $item_type );
+		?>
+		<tr>
+			<td scope="row">
+				<input name="smbd_<?php echo $this->field_slug; ?>_limit" id="smbd_<?php echo $this->field_slug; ?>_limit" value="true" type="checkbox">
+			</td>
+			<td>
+				<?php if ( false === strpos( $this->page_slug, 'remove' ) ) : ?>
```

Comment:
Don't check the condition here. Instead use pass a parameter to this function.

Do it for other functions as well.